### PR TITLE
Identify the names of exported bindings.

### PR DESCRIPTION
**Pull request by rictic**
This is useful for matching up imports to exports. No user facing changes yet.

 - [x] CHANGELOG.md not updated, this isn't exposed anywhere.

### CODE REVIEW

File: src/test/javascript/javascript-export-scanner_test.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
```
Comment (aomarks) [inline src/test/javascript/javascript-export-scanner_test.ts]: 2018

File: src/javascript/esutil.ts [hunk 2]
```diff
@@ -729,3 +730,117 @@ function isStatementWithUniqueStatementChild(node: babel.Node): boolean {
...
+      for (const specifier of declaration.specifiers) {
+        yield specifier.local.name;
+      }
+      break;
+    default:
+      assertNever(declaration);
+  }
+}
+
+// Utility function for getBindingNamesFromDeclaration
```
Comment (aomarks) [inline src/javascript/esutil.ts]: Describe what it does instead of who uses it?

**Comment by aomarks** [comment]
Failing on AppVeyor. Not sure if just a flake, I don't seem to have the power to restart it.

**Comment by rictic** [comment]
The rerun passed, looked like it was a silly npm thing.

**Pull request merged**